src/data/actions/grades.js

```diff
@@ -87,7 +87,7 @@ const fetchGrades = (courseId, cohort, track, showSuccess) => (
   }
 );
 
-const fetchMatchingUserGrades = (courseId, searchText, cohort, track) => (
+const fetchMatchingUserGrades = (courseId, searchText, cohort, track, showSuccess) => (
```

Comment:
I feel like `showSuccess` is not a good name for the variable. 
Can the name be something like `showUpdateSuccessfulBanner`?